public/js/comment.js

```diff
@@ -0,0 +1,26 @@
+const createCommentHandler = async (event) => {
+  event.preventDefault();
+
+  //   Eingabefelder habe ich so ausgelesen
+  const comment = document.getElementById("input-comment").value;
+  const postId = document.getElementById("post-id").textContent;
+  //   Values werden an den Server/API geschickt.
+  //   Mit Fetch wird der Pfad bestimmt, der über api_routes aufgerufen wird; in diesem Fall /post.
+  const response = await fetch("/api/comment", {
+    method: "POST",
+    // Pack die Values in den Body rein.
+    body: JSON.stringify({ comment, postId }),
+    headers: { "Content-Type": "application/json" },
+  });
+  if (response.ok) {
+    document.location.replace("/show-post-comment/" + postId);
+  } else {
+    console.log("New comment failed.", response);
```

Comment:
in place of a console log - consider appending an error message to the page here to notify the user that the comment did not post successfully. if you're using jQuery you could do something like this:
```
$(".container").append("element html here")
```
where the .container is the class or id of the place where you'd like to append the new element.
More information is here:
https://api.jquery.com/append/
:-)